src/test/kotlin/uk/gov/justice/digital/hmpps/nomisuserrolesapi/resource/UserAccountResourceIntTest.kt

```diff
@@ -50,7 +91,7 @@ class UserAccountResourceIntTest : IntegrationTestBase() {
             CreateLocalAdminUserRequest(
               username = "laauser1",
               firstName = "Laa",
-              lastName = "User",
+              lastName = "U'ser",
```

Comment:
Create user tests pulled into inner class to and the create tests (for local/admin/general user) now add in an apostrophe for the "successful" create